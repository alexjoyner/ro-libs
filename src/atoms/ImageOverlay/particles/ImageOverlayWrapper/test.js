import React from 'react';
import { shallow } from 'enzyme';
import { ImageOverlayWrapper } from '.';

describe('ImageOverlayWrapper component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<ImageOverlayWrapper />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
