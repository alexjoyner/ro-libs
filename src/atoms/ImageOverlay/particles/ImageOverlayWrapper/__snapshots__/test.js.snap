// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageOverlayWrapper component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "ImageOverlayWrapper-pf7jti-0",
        "isStatic": true,
        "rules": Array [
          "background:rgba(0,0,0,0.6);height:100%;display:flex;flex-direction:column;justify-content:center;align-items:center;button{&:hover{box-shadow:none;transform:scale(1.05);}&:active{transform:scale(0.99);}}",
        ],
      },
      "displayName": "ImageOverlayWrapper",
      "render": [Function],
      "styledComponentId": "ImageOverlayWrapper-pf7jti-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
